code/game/objects/structures/signs.dm

```diff
@@ -292,3 +292,8 @@
 
 /obj/structure/sign/russian/staff_only
 	icon_state = "RU_staff_only"
+
+/obj/structure/sign/whitedragon
+	name = "White Dragon"
+	desc = "A sign advertising White Dragon, a noodle bar. It's in some sort of ancient language you can't decipher..."
+	icon_state = "neonsign"
```

Comment:
This name is going to be confusing, please name it something to make it obvious that it's for White Dragon.

Reply:
Sorry I meant the iconstate would be confusing, not the actual "name". I'm dumb. Other than that I think this is ready.